### PR TITLE
CASMCMS-7617 Update CMS owned base charts for CASM-2670 changes

### PR DESCRIPTION
**Pull request by ecozzi-hpe**
## Summary and Scope

Update CMS install charts to be in line with CASM-2670 changes

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves CASMCMS-7617

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: charts/cray-import-config/Chart.yaml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+home: https://github.com/Cray-HPE/cray-product-install-charts
+sources:
+  - https://github.com/Cray-HPE/cray-product-install-charts
 maintainers:
 - email: randy.kleinman@hpe.com
   name: rkleinman
-
+annotations:
+  artifacthub.io/images: |
+    - name: cray-cfs
```
Comment (rkleinman-hpe) [inline charts/cray-import-config/Chart.yaml:16]: Copypasta?

File: charts/cray-import-kiwi-recipe-image/Chart.yaml [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
+- https://github.com/Cray-HPE/cray-product-install-charts
 maintainers:
 - email: randy.kleinman@hpe.com
   name: rkleinman
-
+- email: eric.cozzi@hpe.com
+  name: ecozzi-hpe
+annotations:
+  artifacthub.io/images: |
+    - name: cray-cfs
```
Comment (rkleinman-hpe) [inline charts/cray-import-kiwi-recipe-image/Chart.yaml:21]: copypasta?

**Review by rkleinman-hpe — CHANGES_REQUESTED** [review]

File: charts/cray-import-config/Chart.yaml (hunk 1 above)
Comment (ecozzi-hpe) [reply charts/cray-import-config/Chart.yaml]: Fixed

File: charts/cray-import-kiwi-recipe-image/Chart.yaml (hunk 2 above)
Comment (ecozzi-hpe) [reply charts/cray-import-kiwi-recipe-image/Chart.yaml]: Fixed

**Review by jsl-hpe — APPROVED** [review]

**Review by rbak-hpe — APPROVED** [review]

File: charts/cray-import-kiwi-recipe-image/values.yaml [hunk 3]
```diff
@@ -1,30 +1,27 @@
...
 
 # Docker image that will update the product catalog with results of the import
 # of images and recipes. Specifies the product catalog ConfigMap as well.
 catalog:
   image:
-    registry: "dtr.dev.cray.com"
-    repo: cray
-    name: cray-product-catalog-update
+    repository: artifactory.algol60.net/csm-docker/stable/cray-product-catalog-update
     tag: 1.2.50
```
Comment (rkleinman-hpe) [inline charts/cray-import-kiwi-recipe-image/values.yaml:19]: There will be a 1.3.x coming in https://github.com/Cray-HPE/cray-product-catalog/pull/17

**Review by rkleinman-hpe — CHANGES_REQUESTED** [review]

File: charts/cray-import-kiwi-recipe-image/values.yaml (hunk 3 above)
Comment (ecozzi-hpe) [reply charts/cray-import-kiwi-recipe-image/values.yaml:19]: Gottcha. Let me know when we have a version number.

Comment (rkleinman-hpe) [reply charts/cray-import-kiwi-recipe-image/values.yaml:19]: 1.3.1 is building now.

**Review by rkleinman-hpe — APPROVED** [review]

**Comment by rkleinman-hpe** [comment]
We will need an announcement to the product stream people about the changes to the `config_image` and `import_image` keys in the respective `values.yaml` files. I realize this is a backwards incompatible change, but better to proactively communicate that. I will get a CMS announcement out there for this.

**Comment by rkleinman-hpe** [comment]
https://connect.us.cray.com/confluence/display/SCMS/2021-11-19+Cray+Product+Install+Toolset+Breaking+Changes created.

**Pull request merged**